CMakeLists.txt

```diff
@@ -62,6 +62,7 @@ option(NRN_ENABLE_RX3D "Enable rx3d support" ${NRN_ENABLE_RX3D_DEFAULT})
 option(NRN_ENABLE_CORENEURON "Enable CoreNEURON support" ${NRN_ENABLE_CORENEURON_DEFAULT})
 option(NRN_ENABLE_BACKTRACE "Enable pretty-printed backtraces" ${NRN_ENABLE_BACKTRACE_DEFAULT})
 option(NRN_ENABLE_TESTS "Enable unit tests" ${NRN_ENABLE_TESTS_DEFAULT})
+option(NRN_ENABLE_TESTS_BBP "Enable extra tests that require BlueBrain authentication" ${NRN_ENABLE_TESTS_BBP_DEFAULT})
```

Comment:
question - do we need add/expose this BBP specific option here?

Reply:
I think it still works, but it probably doesn't show up in some GUIs. I was just trying to be consistent...